user-guide/en-US/All.asciidoc

```diff
@@ -800,60 +800,122 @@ Just as with the Authorization policy, you can define any number of rules you'd
 ==== IP Whitelist Policy
 [[policy-ip-whitelist]]
 ===== Description
-DESCRIPTION TBD HERE
+The IP Whitelist Policy Type is the counterpart to the IP Blacklist Policy type. In the IP Whitelist policy, only inbound API requests from application clients that satisfy the policy are accepted. 
+
+The IP Blacklist and IP Whitelist policies as complementary, but different, approaches to limiting access to a service:
+* The IP Blacklist policy type is exclusive in that you must specify the IP address ranges to be excluded from being able to access the service. Any addresses that you do not explicitly exclude from the policy are able to access the service.
+* The IP Whitelist policy type is inclusive in that you must specify the IP address ranges to be included to be able to access the service. Any addresses that you do not explicitly include are not able to access the service.
 
 TIP: This is a built-in policy and therefore no plugins need to be installed prior to using it.
 
 ===== Configuration
-CONFIGURATION TBD HERE
+The configuration parameters for an IP Whitelist Policy are:
+
+* IP Address HTTP Header - This is an optional parameter that can be used to provide addition information.
+* Failure Response - The server response code. The possible values for the return code are:
+ ** 500 - Server error
+ ** 404 - Not found
+ ** 403 - Authentication failure
+* IP Addresses - The IP address, and/or ranges of addresses that will be permitted to access the configured API. 
 
 ===== Sample Configuration
-SAMPLE CONFIG TBD
+
 ```json
 {
-   "TBD" : "TBD"
-}
+    "id" : "IPWhitelistPolicy",
+    "policyImpl" : "class:io.apiman.gateway.engine.policies.IPWhitelistPolicy",
+    "name" : "IP Whitelist Policy",
+    "description" : "Only requests that originate from a specified set of valid IP addresses will be allowed through.",
+    "icon" : "thumbs-up",
+    "templates" : [ {
+      "template" : "Only requests that originate from the set of @{ipList.size()} configured IP address(es) will be allowed to invoke the managed API."
+    } ],
+    "formType" : "Default",
+    "deleted" : false
+  }
 ```
 
 
 ==== IP Blacklist Policy
 [[policy-ip-blacklist]]
 ===== Description
-DESCRIPTION TBD HERE
+As its name indicates, the IP blacklist policy type blocks access to an API’s resources based on the IP address of the client application. The apiman Management UI form used to create enables you to use wildcard characters in specifying the IP addresses to be blocked. In addition, apiman gives you the option of specifying the return error code sent in the response to the client if a request is denied. Note that an IP Blacklist policy in a plan overrides the an IP Whitelist policy in the same plan.
```

Comment:
> The apiman Management UI form used to create enables you to use 

Perhaps /used to create/used to configure the policy/

Else, need a noun/pronoun after 'create' I think